XML3-Homework/parrish_1-26_xml3.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<root>
+    <intro> Learning to knit can seem a bit daunting, so if you're keen to kick start your knitting
+        journey but you're not sure where to begin, you’ve come to the right place! We've got loads
+        of brilliant learn to knit videos that cover everything from beginner tutorials on how to
+        hold the needles and knit your first row, through to more advanced knitting techniques such
+        as how to work those tricky button holes. You'll be knitting up a storm in no time! </intro>
+
+    <extraInfo> Getting started Before you begin your journey of learning how to knit, you’ll need a
+        few essentials to get you started. We'll talk you through the essential first steps to
+        getting started including picking a knitting project, choosing the best needles for the job
+        and selecting your yarn, as well as showcasing some fabulous kits. Then enjoy browsing our
+        gallery of step-by-step tutorials. </extraInfo>
+
+    <mainSteps>
+        <step main="first">1. Pick a pattern</step>
+        <extraInfo> The first step on your knitting odyssey is to pick a pattern! We've got hundreds
+            of knitting projects from a host of designers and much-loved brands for all abilities.
+            Whether you're looking to craft a cosy hat for a buddy, a snuggly baby blanket, or
+            dreamy scarf, we have a beautiful pool of projects to inspire you! </extraInfo>
+        <step main="second">2. Choose the best needles to get started</step> First up, if you’re
+        learning traditional knitting techniques, we would recommend starting with single point
+        needles, which are used in our beginner videos. But there’s a whole menagerie of knitting
```

Comment:
@kzp308 But there's a lot of "floating" text here that seems like it's connected to that first step you've closed. That first step was about choosing knitting needles. And this part I'm reading seems to be about how to make that decision--how to choose knitting needles.Maybe this information can be grouped *inside* the step it's associated with? Like this maybe:

```xml
<step main="second">2. Choose the best needles to get started. 
          <note> First up, if you're learning traditional knitting techniques, we would recommend starting with single 
              point needles, which are used in our beginner videos. But there’s a whole menagerie of knitting
              needles to explore from DPNs to circular in different sizes and materials, from bamboo to plastic.
           </note>
      </step>
```

See how that's nesting the related information?